packages/edit-post/src/components/preferences-modal/index.js

```diff
@@ -116,6 +116,15 @@ export default function EditPostPreferencesModal() {
 									label={ __( 'Show block breadcrumbs' ) }
 								/>
 							) }
+							<EnableFeature
+								featureName="allowRightClickOverrides"
+								help={ __(
+									'Allows contextual menus via right-click, overriding browser defaults.'
```

Comment:
Do you think we should specify that the contextual menus display in the list view?

Reply:
At the moment, I was thinking of leaving it as-is, because of the potential to add right-click behaviour to the editor canvas in follow-ups, where we'll re-use the setting. Happy to change it to specify list view for now, if it's confusing, though!

Reply:
Updated to include `list view`:

<img width="518" alt="image" src="https://github.com/WordPress/gutenberg/assets/14988353/562f9026-8b70-4fb1-82ba-df8c67483bfa">